### PR TITLE
Fix a probable typo in compositemap

### PR DESCRIPTION
**Pull request by kathivashkiv**

### CODE REVIEW

File: sunpy/map/compositemap.py [hunk 1]
```diff
@@ -302,7 +302,7 @@ def draw_limb(self, index=None, axes=None, **kwargs):
         return self._maps[index].draw_limb(axes=axes, **kwargs)
 
     @u.quantity_input
-    def draw_grid(self, index=None, axes=None, grid_spacing: u.deg = 20*u.deg, **kwargs):
+    def draw_grid(self, index=None, axes=None, grid_spacing: u.dex = 20*u.dex, **kwargs):
```
Comment (nabobalis) [inline sunpy/map/compositemap.py:305]: I am not sure this is a typo. u.deg should stand for degrees.

Comment (kathivashkiv) [reply sunpy/map/compositemap.py:305]: That is what I wanted to ask. I've run into strange thing while studying code related to issue #4492 (Passing draw_grid=True to CompositeMap.peek is broken). In compositemap.py, line 305. My pylint said that "There is no "deg" member, maybe "dex".
I tried to find "deg" in related astropy/units, but couldn't. Sorry, if I violated the procedure of PR

Comment (kathivashkiv) [reply sunpy/map/compositemap.py:305]: Should I then close/delete this request?

Comment (nabobalis) [reply sunpy/map/compositemap.py:305]: I would say that this isn't a typo.

**Comment by ayshih** [comment]
@kathivashkiv Thanks for this PR, but `u.deg` is correct and not a typo, so I'm going to close it.

**Pull request closed without merging**